src/api/features.api.js

```diff
@@ -44,9 +44,9 @@ export class SelectableFeature extends EventEmitter {
         super()
         this._id = id
         // using the setter for coordinate (see below)
-        this._coordinates = coordinates
-        this._title = title
-        this._description = description
+        this.coordinates = coordinates
+        this.title = title
+        this.description = description
```

Comment:
Why the change? Those properties are exposed as `instance.coordinates|title|description` through the fact that there is a getter set for them.
If you want to remove the underscored part, I don't mind, but you have to make sure any internal use below doesn't use this underscored name anymore

Reply:
The changes is needed due to the setter, otherwise the setter is not called. For the coordinates the setter make sure to encapsulate single coordinate in an array, which was the bug because before when passing a single coordinate not in a array it was not correct and a geojson object selection was not marked as selected. I did not fully understand why we need to encapsulate single coordinate in an array though. Would be great to simplify the code by removing this setter and make sure to always path coordinate as follow: `[[x, y], ...]`. But that would require probably much more rework and effort.